pkg/app/ops/insightcollector/deployments.go

```diff
@@ -93,6 +93,10 @@ func (c *Collector) findDeploymentsCreatedInRange(ctx context.Context, from, to
 					Field:     "CreatedAt",
 					Direction: datastore.Desc,
 				},
+				{
```

Comment:
@Hosshii you may need to update firestore index as well to enable this query to work. Firestore indexes stored under `pkg/app/ops/firestoreindexensurer/indexes.json`

Reply:
Ok, thank you!